tests/integration/components/two-way-data-binding-test.js

```diff
@@ -52,6 +52,36 @@ describeComponent(
           expect(this.blur).to.equal('dodge');
         });
       });
+    });
+
+    describe("default value of null with two way data binding disabled", function() {
+      beforeEach(function() {
+        this.set('make', null);
+        this.set('selectAction', (value) => {
+          this.set("make", value);
+          this.set("wasCalled", true);
+        });
+
+        this.render(hbs`
+          {{#x-select value=make one-way=true action=(action selectAction)}}
+            {{#x-option value="ford" class="spec-ford-option"}}Ford{{/x-option}}
+            {{#x-option value="chevy"}}Chevy{{/x-option}}
+            {{#x-option value="dodge" class="spec-dodge-option"}}Dodge{{/x-option}}
+          {{/x-select}}
+        `);
+      });
+
+      it("displays the first item in the list", function() {
+        expect(this.$('select option:selected').text()).to.equal("Ford");
+      });
+
+      it("sets the default value to the first element", function() {
+        expect(this.get('make')).to.equal("ford");
```

Comment:
I'm going to guess there's a difference in `"Ford"` & `"ford"`. This is partly my fault since I made the the initial test file but maybe we should make the value keys something more explicit. 

`{{#x-option value="fordKey" class="spec-ford-option"}}Ford{{/x-option}}`? 

Or after typing all that maybe just a comment that says lowercase ford is significant because it's the value of the option, not the display key.